09-Multiparticle-Collision-Dynamics/02-Diffusion-in-Solution.ipynb

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
<p>"Simple" is a judgement we cannot make for the reader. The first sentence should read: "Apply the <code>Bulk</code> streaming method to propagate solvent particles with no confining geometry." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>